config/locales/es/about.es.yml

```diff
@@ -0,0 +1,20 @@
+es:
+  about:
+    title: 'Acerca de REFUGE'
+    p1header: '¿De qué se trata REFUGE restrooms?'
+    p1: 'REFUGE es una aplicación por internet que busca proveer acceso seguro a baños para personas trans, intersexo, y personas género no conformes. Cuando el sitio web Safe2Pee cesó de funcionar, dejó un vacio en nuestros corazones. REFUGE continúa el proyecto de Safe2Pee y vuelve a poner a disposición ese recurso valorado para quienes necesitan un espacio seguro para usar el baño. Es posible buscar baños por proximidad a una ubicación, agregar nuevos sanitarios a la lista, además de comentar y agregar un puntaje a los baños listados. Queremos crear una comunidad enfocada en más que solo buscar baños seguros, que sientan que pueden proyectar y participar en crear espacios para apoyar a personas trans, intersexo y personas género no conformes.'
```

Comment:
I think the about page might have changed from when this PR was originally created. This now has multiple sections and is displaying English since it doesn't match: https://github.com/RefugeRestrooms/refugerestrooms/blob/develop/config/locales/en/about.en.yml#L6